pandas/core/computation/eval.py

```diff
@@ -159,16 +160,16 @@ def _check_for_locals(expr: str, stack_level: int, parser: str):
 
 def eval(
     expr,
-    parser="pandas",
+    parser: str = "pandas",
```

Comment:
@simonjayhawkins IIUC we can't use Literal yet.  what if we used enums as a stopgap

```
class ParserStr(enum):
    pandas = "pandas"
    numexpr = "numexpr"
```

that way we could start filling these in where appropriate and when we can use Literal swapping them out will be easy (wont require hunting them down)

Reply:
i think we are now using Literal